Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -346,8 +350,14 @@ WarpX::computePhiCartesian (const amrex::Vector<std::unique_ptr<amrex::MultiFab>
 
     // Solve the Poisson equation
     linop.setDomainBC( lobc, hibc );
+
+    // This is apparently necessary for how AMReX expects phi
+    for (int lev=0; lev < rho.size(); lev++){
+        phi[lev]->mult(-1.*PhysConst::ep0);
+    }
```

Comment:
It would be best to fold this into the assignment expression, so that the data don't have to be streamed over twice